tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "lib": ["esnext"],
```

even though these are node scripts, playwright interacts with the DOM APIs
```suggestion
    "lib": ["esnext", "DOM"],
```

---

fixed on my local.